src/gourmand/importers/web_importer.py

```diff
@@ -42,7 +48,19 @@ def import_urls(urls: List[str]) -> Tuple[List[str], List[str]]:
             urls.remove(url)
 
     for url in urls:
-        recipe = scrape_html(html=None, org_url=url)
+        req = Request(url, headers=HEADERS)
+        try:
+            response = urlopen(req)
+            html = response.read()
+        except URLError as e:
+            if hasattr(e, 'reason'):
```

Comment:
Are there really URLError instances with different properties? According to https://docs.python.org/3/library/urllib.error.html#module-urllib.error, `URLError.reason` will always be available, thus we can access it directly and do not need these more or less odd attribute checks.

By the way: Please prefer logging over plain print statements.